pom.xml

```diff
@@ -19,4 +19,13 @@
     <maven.compiler.target>14</maven.compiler.target>
```

Comment:
You guys provided SNAPSHOT version for the project. I suppose that you have ideas why and how we use that. If not, please ask. Don't need to spend time investigate this.

Reply:
I think that the SNAPSHOT version is to note the times of release, plz tell me if there is something else or my knowledge is not true

Reply:
This was automatically generated when I used an extension to create the maven project. Having no idea about this, could you provide us information about this.

Reply:
got it. I'll share this in next meeting